http-client/src/main/java/io/avaje/http/client/HttpException.java

```diff
@@ -75,15 +75,15 @@ public HttpException(int statusCode, Throwable cause) {
   }
 
   HttpException(HttpResponse<?> httpResponse, DHttpClientContext context) {
-    super();
+    super("Http Call failed with status: " + httpResponse.statusCode());
```

Comment:
I think I'll change that to "Http call ..." using lower case "c" for "call" as we have `HttpCall` as a type